remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -20,9 +20,10 @@
 # THE SOFTWARE.
 #
 
-if [[ $# -eq 1 ]]
+if [[ $# -eq 1 ]] || [[ $# -eq 2 ]]
 then
```

Comment:
using -lt is better 2)how about only 1 parameter passing, then variable BINARY_LOCATION is null or empty which might cause problem